src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -8,14 +8,180 @@ $border-radius: 28px;
   display: flex;
   flex-direction: row;
   margin-top: 20;
+  flex: 1;
 
   .party-map-container {
-    width: 80%;
+    height: calc(100vh - #{$navbar-height});
+    flex: 1;
+    overflow: scroll;
+    &::-webkit-scrollbar-track {
+      border-radius: 3px;
+      background-color: rgba(0, 0, 0, 0);
+    }
+
+    &::-webkit-scrollbar {
+      width: 15px;
+      height: 15px;
+      background-color: rgba(0, 0, 0, 0);
+    }
+
+    &::-webkit-scrollbar-thumb {
+      border-radius: 4px;
+      background-color: rgba(255, 255, 255, 0.3);
+    }
+    &::-webkit-scrollbar-corner {
+      background-color: transparent;
+    }
 
-    .party-map-content {
-      flex: 1;
+    .party-map-zoom-container {
       display: flex;
-      margin: 20px auto 20px auto;
+      flex-direction: column;
+      background-color: rgba(0, 0, 0, 0.5);
+      border-radius: 22px;
+      justify-content: center;
+      position: absolute;
+      bottom: 20px;
+      width: 40px;
```

Comment:
I dont think we should be using any fixed values for dimensions if there is ever an intention to make the app fluid/responsive.

For such a "small" width use something like:

`width: 2.5rem` which would in this case (font-size default: 16px) be 40px

Reply:
the best solution for truly fluid apps would be to use `vw` and `vh` units but that requires a lot of reworks.

With the use of `rem` if any person has their default font size changed to be bigger (impaired vision) the map zoom container will be bigger as well